public/js/jobManager.js

```diff
@@ -14,6 +14,10 @@ function buildDownloadLink(jobId, state) {
     + '" href="' + url + '">' + msg + '</a>';
 }
 
+function buildResultLink(jobId) {
+  return '<a href="/result.html?api_key=' + getParams()['api_key'] + '&job_id=' + jobId + '">Visualiser les résultats</a>'
```

Comment:
Could this open a new page rather than changing current one ?

Reply:
Actually i'm in school so i can't change it until tomorrow
But you can 'middle click' on the link to open it on a new page

Reply:
Yes, I was talking about changing default behaviour :)